cmd/livepeer/livepeer.go

```diff
@@ -378,20 +379,25 @@ func main() {
 			return
 		}
 
-		client, err := eth.NewClient(ethcommon.HexToAddress(*ethAcctAddr), keystoreDir, backend, ethcommon.HexToAddress(*ethController), EthTxTimeout)
+		var bigMaxGasPrice *big.Int
+		if *maxGasPrice > 0 {
+			bigMaxGasPrice = big.NewInt(int64(*maxGasPrice))
+		}
+
+		client, err := eth.NewClient(ethcommon.HexToAddress(*ethAcctAddr), keystoreDir, *ethPassword, backend, ethcommon.HexToAddress(*ethController), EthTxTimeout, bigMaxGasPrice)
 		if err != nil {
-			glog.Errorf("Failed to create client: %v", err)
+			glog.Errorf("Failed to create Livepeer Ethereum client: %v", err)
 			return
 		}
 
 		var bigGasPrice *big.Int
 		if *gasPrice > 0 {
+			glog.Warning("-gasPrice is deprecated, you can use -maxGasPrice instead")
 			bigGasPrice = big.NewInt(int64(*gasPrice))
 		}
 
-		err = client.Setup(*ethPassword, uint64(*gasLimit), bigGasPrice)
-		if err != nil {
-			glog.Errorf("Failed to setup client: %v", err)
+		if err := client.SetGasInfo(uint64(*gasLimit), bigGasPrice); err != nil {
+			glog.Errorf("Failed to setup Livepeer Ethereum Client: %v", err)
```

Comment:
Looks like an old error message is used here. Let's update it to be more descriptive of the error condition.

Reply:
I felt like this was most descriptive as it's mostly old code repurposed from `client.Setup`. 

It doesn't only set up the gas info but also binds the options to the contract. 

Maybe "Failed to set up contract transactors", but that seemed geth specific lingo.